### DISCUSSION TITLE
Allow to change Nextcloud Talk STUN and TURN domains persistently

### DISCUSSION THREAD

**Discussion by ShadyNawara** [nextcloud/all-in-one#1088] [Ideas] [answered]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Steps to reproduce
1.go to settings > Talk
2.change stun and turn server domain
3.restart images

### Expected behavior 
new server domain should be used and persist across server restarts
### Actual behavior 
domain name settings are not applied instead the original nextcloud domain is used

### Host OS 
MacOS 12.5.1

#### Nextcloud AIO version 
v1.7.1

#### Current channel 
latest

#### Other valuable info 
I am using a cloudflare tunnel pointing to the apache container on port 11000 following the reverse-proxy guide. This works great except for the talk service. I need to be able to use a different subdomain with port forwarding setup for use with talk while keeping the cloudflare subdomain for the rest of nextcloud. 

I think the tunserver.conf file should be mounted from a volume instead of inside the container to allow for this setting to persist across reboots

**Comment by szaimen** [comment]
This is not a bug but a feature request. Moving to discussions.

**Comment by STBoyden** [comment]
I would also like this feature, would be useful in the case of the main Nextcloud domain being behind Cloudflare, as is mentioned in the original post.

**Comment by szaimen** [comment, marked as answer]
This will get addressed with https://github.com/nextcloud/all-in-one/pull/1307

**Reply by szaimen** [reply]
This is now released with v2.1.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel
